src/backend/mpi/swdsm.h

```diff
@@ -368,6 +391,23 @@ unsigned long get_classification_index(uint64_t addr);
  */
 bool _is_cached(std::size_t addr);
 
+/* CSPext: Node rebuild function */
+void redundancy_rebuild(argo::node_id_t node);
+
+/* CSPext: Create or re-create globalDataWindow */
+/**
+ * @brief Check flag in the table and update MPI window.
+ * @param tbl Pointer to table.
+ * */
+void node_alternation_table_recreate_globalDatawindow(node_alternation_table *tbl);
```

Comment:
A shorter name would be nice, such as "node_alt_tbl_recreate_globalDatawindow". Still long, but not sure how to shorten it further without shortening the variable name too.